### PR TITLE
v3.2 release notes

### PR DESCRIPTION
**Pull request by DominikaK**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | n/a
| Versions      | all

### CODE REVIEW

File: docs/releases/ibexa_dxp_v3.2.md [hunk 1]
```diff
@@ -23,6 +23,8 @@ including eCommerce administration.
 
 ### Autosave
 
+eZ Platform can now save your edits in a Content item or product automatically to help you preserve the progress in an event of a failure. For more information, see [Autosave](../publishing/publishing.md#autosave).
```
Comment (DominikaK) [inline docs/releases/ibexa_dxp_v3.2.md:26]: Link to user guide, needs a full address.

File: docs/releases/ibexa_dxp_v3.2.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+![New Content structure]()
+
+![Commerce administration]()
+
+### DAM connector
+
+### Autosave
+
+eZ Platform can now save your edits in a Content item or product automatically to help you preserve the progress in an event of a failure. For more information, see [Autosave](../publishing/publishing.md#autosave).
```
Comment (dabrt) [inline docs/releases/ibexa_dxp_v3.2.md:26]:
```suggestion
eZ Platform can now save your edits in a Content item or product automatically to help you preserve the progress in an event of a failure. For more information, see [Autosave](https://doc.ezplatform.com/projects/userguide/en/latest/publishing/publishing/#autosave).
```

File: docs/releases/ibexa_dxp_v3.2.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
+# Ibexa DXP v3.1
```
Comment (SylvainGuittard) [inline docs/releases/ibexa_dxp_v3.2.md:1]:
```suggestion
# Ibexa DXP v3.2
```

File: docs/releases/ibexa_dxp_v3.2.md [hunk 4]
```diff
@@ -88,6 +88,14 @@ Also, you can now apply new filters that are intended to help you declutter the
 
 For more information, see [Calendar widget](https://doc.ibexa.co/projects/userguide/en/latest/publishing/advanced_publishing_options/#calendar-widget).
 
+### Cloning content types
```
Comment (DominikaK) [inline docs/releases/ibexa_dxp_v3.2.md:91]:
```suggestion
### Cloning Content Types
```

**Pull request merged**